go.mod

```diff
@@ -3,13 +3,16 @@ module ai-developer
 go 1.22.3
 
 require (
+	github.com/creack/pty v1.1.21
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/gin-contrib/zap v1.1.3
 	github.com/gin-gonic/gin v1.10.0
 	github.com/go-git/go-git/v5 v5.12.0
 	github.com/goccy/go-json v0.10.2
 	github.com/google/go-github v17.0.0+incompatible
 	github.com/google/uuid v1.4.0
 	github.com/googollee/go-socket.io v1.7.0
+	github.com/gorilla/websocket v1.5.3
```

Comment:
why gorialla we had preferred socketio over it

Reply:
https://github.com/googollee/go-socket.io we had moved ahead with socketio for better lifecycle and rejected this @czezi